addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -18,6 +18,32 @@
         </xpath>
     </template>
 
+    <template id="product_cart_lines" inherit_id="website_sale.cart_lines" name="Wishlist Button">
```

Comment:
I find the view name misleading as there are other wishlist buttons; better remove it.
```suggestion
    <template id="product_cart_lines" inherit_id="website_sale.cart_lines">
```